src/Libraries/Display/Display.cs

```diff
@@ -112,13 +129,63 @@ public static Display BySurfaceColors(Surface surface, Color[][] colors)
             {
                 if (list.Count() != size)
                 {
-                    throw new ArgumentException("The list of colors must not be a jagged list.");
+                    throw new ArgumentException(Resources.JaggedListExceptionMessage);
                 }
             }
 
             return new Display(surface, colors);
         }
 
+        /// <summary>
+        /// Create a colored mesh using points and colors.
+        /// 
+        /// The list of points supplied is used to construct a triangulated mesh, with
+        /// non-joined vertices.
+        /// </summary>
+        /// <param name="points">A list of Points. 
+        /// 
+        /// Only triangular meshes are currently supported. Each triplet of points in the list will form one 
```

Comment:
I think that this is okay, and makes using the node much simpler, but I guess we'll need to evaluate if this node is performant enough for the datasets that need to be visualized since we'll be repeating vertex information and this requires creating many duplicate Autodesk.DesignScript.Geometry.Points objects.